### ISSUE TITLE
Music stop playing when de/connecting bluetooth device

### ISSUE THREAD

**Issue by brenard** [segler-alex/RadioDroid#50]
Hi, 

When music is playing and I connect an audio bluetooth device, music stop playing and I have to and play music again to heard music on bluetooth audio output. The same happening when I disconnect bluetooth device : I have to stop and play music again to heard music on phone speaker.

Thank for this great FROS app !

**Comment by segler-alex** [comment]
thats interesting, because, there is another bug #22 , that says the oposite thing :)
i did not implement something that stops the streaming. what mobile phone do you have?

**Comment by tanty** [comment]
It doesn't happen to me with 0.24

**Comment by glitchphan** [comment]
Yeah, I'm listening to wired headphones right now. I disconnected them and the station keeps playing. Same when I reconnected.

**Comment by brenard** [comment]
Hi, I try again with version 0.24 and it's not working for me : I start listening radio on my phone speaker and when I connect it to Bluetooth speaker, radio it's still playing on UI but I don't hear it on bluetooth speaker. If I stop radio and play it again, music work great on bluetooth speaker.
